src/meta/processors/partsMan/ListHostsProcessor.cpp

```diff
@@ -74,7 +74,7 @@ Status ListHostsProcessor::allHostsWithStatus() {
     }
 
     // get hosts which have send heartbeat recently
-    auto activeHosts = ActiveHostsMan::getActiveHosts(kvstore_, FLAGS_heartbeat_interval_secs + 1);
+    auto activeHosts = ActiveHostsMan::getActiveHosts(kvstore_, FLAGS_heartbeat_interval_secs * 2);
```

Comment:
Actually I'm not sure why heartbeat_interval_secs * 2 after talk with dooble offline, maybe more safe

Reply:
Yes,  the purpose is to make the active hosts  returned more stable.